### PR TITLE
Remove out-dated macros

### PR DESCRIPTION
**Pull request by yunline**
Issue: https://github.com/pygame-community/pygame-ce/issues/2077

### CODE REVIEW

**Review by MyreMylar — APPROVED** [review]
LGTM 👍 

You can see these flags here: https://wiki.libsdl.org/SDL2/SDL_WindowFlags

And all should be present in our minimum SDL version, I would expect these defines to do nothing at all so it seems fine to remove them.

File: src_c/include/pgcompat.h [hunk 1]
```diff
@@ -29,79 +29,7 @@ typedef uint8_t Uint8;
...
-#define SDL_WINDOW_TOOLTIP 0
-#endif
-
-#ifndef SDL_WINDOW_POPUP_MENU
-#define SDL_WINDOW_POPUP_MENU 0
-#endif
-
-#ifndef SDL_MOUSEWHEEL_FLIPPED
-#define NO_SDL_MOUSEWHEEL_FLIPPED
-#endif
```
Comment (Starbuck5) [inline src_c/include/pgcompat.h:104]: Unlike the window flags one, this may still be used.

**Comment by Starbuck5** [comment]
I guess the `#ifndef` doesn't work because they are not macros, they're enum values? So the defines are then shadowing the real values.

Added a note about one we may want to keep.

Just fyi it's spelled "macros."

File: src_c/include/pgcompat.h (hunk 1 above)
Comment (yunline) [reply src_c/include/pgcompat.h]: Solved

**Pull request merged**